dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -46,14 +46,17 @@ sources:
             tests:
               - not_null:
                   name: iasworld_dweldat_attic_not_null
+                  select_columns: &select-columns ["taxyr", "parid", "card", "who", "wen"]
                   # Excludes non-regression classes, as defined in Assessor's
                   # online PDF of all class definitions
                   config: &unique-conditions
-                    where: | 
+                    where: |
                       CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
                       AND cur = 'Y'
                       AND deactivat IS NULL
                       AND class NOT IN ('201', '213', '218', '219', '220', '221', '224', '225', '236', '240', '241', '290', '294', '297')
+                  meta:
+                    description: attic should not be null
```

Comment:
**issue:** I think it's important to add labels to most of the `dweldat` tests indicating what the underlying column is, since most people won't be familiar with the DB column names. This is especially true for `user` columns. I suggest something like:

```suggestion
                    description: attic (attic type) should not be null
```

Reply:
Makes sense! I went through the file and added labels to most of the dweldat columns in 7ab148ef5729656b368e5e35515ce3a6513a358d.